picard/cluster.py

```diff
@@ -475,7 +475,7 @@ def add(self, word):
         index, count = self.words[word]
         if index == -1:
             token = self.tokenize(word)
-            if token == '':
+            if token == '':  # nosec
```

Comment:
Which warning was it printing??

Reply:
Bandit B105: Test for use of hard-coded password strings

It checks for variable name "token". We could rename the variable also, but it is fitting in this case. It's kind of a stupid check, really.